arch/arm/dts/crocodile.dtsi

```diff
@@ -382,7 +382,7 @@
 	status = "okay";
 };
 
-/* LTE module */
+/* MUC/stm32 coprocessor */
```

Comment:
Changes to this file should come from https://github.com/hexagon-geo-surv/linux-leica/blob/leica/linux-5.19.y/arch/arm64/boot/dts/freescale/crocodile.dtsi

Reply:
ACK - spawned: https://github.com/hexagon-geo-surv/linux-leica/pull/86